src/utils/env.js

```diff
@@ -0,0 +1,19 @@
+// Use relative import to resolve during nuxt runtime
+import stringToBoolean from './string-to-boolean'
+
+/**
+ * Default environment variables are set on this key. Defaults are fallbacks to existing env vars.
+ * All boolean values should be designed to be false by default.
+ */
+export const env = {
```

Comment:
I wonder if there is a performance boost to using `Object.freeze` on this object literal?

Reply:
Maybe? I doubt it's a measurable difference though.